src/backends/torch/torchlib.cc

```diff
@@ -409,6 +410,9 @@ namespace dd
         _module._loss_id = 0;
         _module._linear_in = 1;
       }
+    else if (_template == "detr")
```

Comment:
Remove

Reply:
No this is required unfortunately, otherwise execution falls within the last `else` statement and fails under unknown / empty template.

Reply:
ok, my bad